src/team/trajectory.py

```diff
@@ -64,7 +64,10 @@ def joint_timestamps(self) -> np.ndarray:
 
     @property
     def average_sampling(self) -> float:
```

Comment:
The comment about window is based on the computation of average sampling, here is the sampling frequency in Hz right?

Reply:
Yes, i don't think I changed the old formulation :). I just made it independent on start and end time.

Reply:
I have chaged this method to return the period :). IMO it's easier to use because we were always converting Hz to sec